db/migrate/20170320220020_add_preferred_email_format.rb

```diff
@@ -0,0 +1,8 @@
+require "migrate"
+
+class AddPreferredEmailFormat < ActiveRecord::Migration
+  def change
+    create_enumeration :email_format_enum, %w(text_only multipart)
```

Comment:
I'd suggest using the actual MIME types here - so `text/plain` and `multipart/alternative` or does that cause problems with what postgres allows for enumeration names? If so then maybe just replace the slash with an underscore?